airflow/models/dagrun.py

```diff
@@ -349,13 +354,15 @@ def get_previous_scheduled_dagrun(self, session: Session = None):
         ).first()
 
     @provide_session
-    def update_state(self, session: Session = None) -> List[TI]:
+    def update_state(self, session: Session = None, handle_callback: Optional[bool] = True) -> List[TI]:
         """
         Determines the overall state of the DagRun based on the state
         of its TaskInstances.
 
         :param session: Sqlalchemy ORM Session
         :type session: Session
+        :param handle_callback: To run Dag Callbacks or not
```

Comment:
```suggestion
        :param handle_callback: Should dag callbacks (success/failure, SLA etc) be invoked directly (true),
            or recorded as a pending request in the ``callback`` property
```

Reply:
fixed in https://github.com/astronomer/airflow/pull/913/commits/29af60d324509c46f983559b5ab6d702a42eebee